src/velero/values/registry1-values.yaml

```diff
@@ -14,3 +14,11 @@ initContainers:
     volumeMounts:
       - mountPath: /target
         name: plugins
+  - name: velero-plugin-for-csi
+    image: registry1.dso.mil/ironbank/opensource/velero/velero-plugin-for-csi:v0.7.0
+    imagePullPolicy: IfNotPresent
+    volumeMounts:
+      - mountPath: /target
+        name: plugins
+configuration:
+  features: EnableCSI
```

Comment:
Could you resolve the handful of linting issues here (just newlines at the end of the files) + merge the latest main changes in?